skimage/filters/tests/test_unsharp_mask.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from skimage.filters import unsharp_mask
+import pytest
+
+
+@pytest.mark.parametrize("shape,multichannel",
```

Comment:
Also, please, do not use `pytest` directly. Instead, import `parametrize` from `skimage._shared.testing` (our thin wrapper) (as in https://github.com/scikit-image/scikit-image/blob/master/skimage/util/tests/test_dtype.py, for example).